lambda/app/src/utils/constants.ts

```diff
@@ -1,2 +1,15 @@
 export const MS_GRAPH_URL = 'https://graph.microsoft.com';
 export const CYPRESS_MOCKED_IDIR_LOOKUP = [{ mail: 'pathfinder.ssotraining2@gov.bc.ca', id: 1 }];
+export const bcscIdpMappers = [
+  { name: 'username', type: 'oidc-username-idp-mapper', template: '${CLAIM.sub}@${ALIAS}' },
+];
+export const bcscClientScopeMappers = [
```

Comment:
is bcscClientScopeMappers needed? if not can we remove it?

Reply:
oh yeah ill take that out, no longer needed